src/openrct2/world/Banner.h

```diff
@@ -22,22 +22,28 @@
 
 #define BANNER_NULL 255
 #define MAX_BANNERS 250
-#define BANNER_INDEX_NULL -1
+#define BANNER_INDEX_NULL ((BannerIndex)-1)
```

Comment:
Perhaps a good moment to use a C++ constexpr here?

Reply:
To my surprise, `MAX_BANNERS` can be made constexpr too, even while it's used for array sizes! I'm sure I bumped into a problem with these before, but that must have been an older C++ version then.